### ISSUE TITLE
Replacing the logic with MemoryReactiveStore with NYTimes/Store

### ISSUE THREAD

**Issue by LOG-TAG** [n26/N26AndroidSamples#2]
Thanks for this sample, very informative!!, I'm just thinking of this as an enhancement for this sample, instead of keeping separate coding caching etc why can't we simply use https://github.com/NYTimes/Store any feedback on this? :)

**Comment by luciapayo** [comment]
Yes, actually the idea is that you can use whatever you want in the data layer as long as it follows the reactive pattern. 
I want to try out NYTimes store and also Room in the future :)

**Comment by Shakezulla57** [comment]
Hi Lucia, thanks for the great example. Do you have anything in the works that uses Room with the `ReactiveStore`. A Room `dao` might have very specific sql queries to return data, that don't necessarily align with the methods in the `ReactiveStore` interface. Am I thinking about this wrong?
